sql/src/main/java/io/crate/operation/fetch/FetchCollector.java

```diff
@@ -59,7 +59,8 @@
         this.streamers = streamers;
         this.readerContexts = searcher.searcher().getIndexReader().leaves();
         this.fieldsVisitor = new CollectorFieldsVisitor(this.collectorExpressions.length);
-        CollectorContext collectorContext = new CollectorContext(mapperService, indexFieldDataService, fieldsVisitor, readerId);
+        CollectorContext collectorContext = new CollectorContext(
+            mapperService, indexFieldDataService, fieldsVisitor, readerId, (byte)0);
```

Comment:
why is the relation hardcoded here? shouldnt we use the relation id of the fetch source?